src/solace_ai_connector/solace_ai_connector.py

```diff
@@ -120,9 +119,9 @@ def setup_logging(self):
         """Setup logging"""
         log_config = self.config.get("log", {})
         stdout_log_level = log_config.get("stdout_log_level", "INFO")
-        file_log_level = log_config.get("file_log_level", "DEBUG")
+        log_file_level = log_config.get("log_file_level", "DEBUG")
         log_file = log_config.get("log_file", "solace_ai_connector.log")
-        setup_log(log_file, stdout_log_level, file_log_level)
+        setup_log(log_file, stdout_log_level, log_file_level)
```

Comment:
Every single place we have been using this variable was using the wrong name, so I just renamed here instead of updating every single yaml file. Could be breaking